### DISCUSSION TITLE
Stage 2 simulations

### DISCUSSION THREAD

**Discussion by justinkirion** [reach-lab/MixWildGUI#165] [Help Wanted] [answered]
Hello, 

It's me again. I am wondering if there is a way to get the different datasets that are simulated in stage 2 to compute model estimators.

**Comment by hedeker** [comment, marked as answer]
an interesting question.  those datasets are not saved unfortunately.  however, they are random draws from the subjects empirical Bayes (EB) estimates of the random effects, and these EB estimates are saved in a file with the suffix of "_ebvar"   That file contains each subjects ID, the empirical Bayes estimates and the posterior variance-covariance matrix associated with the EB estimates.  The latter is in packed form.  so, if you have a model with 1 random location and 1 random scale effect, then the "_ebvar" file contains:
ID, random location, random scale,  posterior variance for random location, posterior covariance, posterior variance for random scale

you can then create the simulated datasets by taking random draws from a normal distribution for each person using the EB estimates as the mean and the posterior var-covar as the variance matrix.  i realize this takes some work.

**Reply by justinkirion** [reply]
Thanks for your clear answer ! :)
